lms/templates/login.html

```diff
@@ -133,7 +141,7 @@
        if (pipeline_running) {
          $('#login-form').submit();
        }
-    })('${pipeline_running | n, js_escaped_string}')
+    })(${pipeline_running | n, dump_js_escaped_json})
```

Comment:
Initially it was without a string quotes like before the last commit. In one of my earlier PR before i moved it here i did the same thing and it was not needed and you did pointed me to fix it. https://github.com/edx/edx-platform/pull/20633#discussion_r286100406 . In the original code before the last change there was no quotes and i dont know why it was changed. So i just tried to moved it back to original state. In either case the code is using the variable as boolean inside if conditional. It would work both ways.

Reply:
I'd say closer to the original code sounds right.  Thanks for explaining.